disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +208,13 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
```

Comment:
I think the backoff function should be in disco_aws_util.py or some general utility module